src/components/pages/Login/Login.js

```diff
@@ -46,35 +46,37 @@ function Login({ setUser, ...props}) {
     }
     // send request to server
     const receivedUser = await loginHandler({
-        username: state.username,
-        password: state.password
+      username: state.username,
+      password: state.password
     });
 
     // if the response has token => successful login
     if (receivedUser.hasOwnProperty('token')) {
-        setUser(getUser());
+      toast.success('Logged in successfully');
 
-        props.history.push('/');
+      setUser(getUser());
+
+      props.history.push('/');
     } else {
-        // on unsuccessful login, render server error message
-        updateState(prevState => ({
-            ...prevState,
-            errors: [{ error: receivedUser.message }],
-            loading: false
-        }))
+      // on unsuccessful login, render server error message
+      updateState(prevState => ({
+        ...prevState,
+        errors: [{ error: receivedUser.message }],
```

Comment:
Just to have consistency, I think would be a great idea to use toast for the errors too 👍

Reply:
Will be changed in different PR for all the forms